libopae/plugins/xfpga/properties.c

```diff
@@ -57,9 +57,8 @@ xfpga_fpgaGetPropertiesFromHandle(fpga_handle handle, fpga_properties *prop)
 	return result;
 }
 
-
-fpga_result __FPGA_API__
-xfpga_fpgaGetProperties(fpga_token token, fpga_properties *prop)
+fpga_result __FPGA_API__ xfpga_fpgaGetProperties(fpga_token token,
+						 fpga_properties *prop)
 {
 	struct _fpga_properties *_prop;
 	fpga_result result = FPGA_OK;
```

Comment:
Should we call the API `fpgaGetProperties` with a null token to allocate the properties structure?

Reply:
Solved by commit 'Fix code review comments.'